### DISCUSSION TITLE
Chromium  *force-device-scale-factor*

### DISCUSSION THREAD

**Discussion by spacrypto** [hyprwm/Hyprland#5458] [Q&A]
![20240406_15h20m48s_grim](https://github.com/hyprwm/Hyprland/assets/120880128/d78699ad-5063-4679-8d70-450d5a4ed9ca)

Hello friends! I couldn’t understand why, when installing force-device-scale-factor in my chromium, Hyprland doesn’t stretch the chromium to the full screen! There is no such problem in Sway (window manager)!
Example
chromium --force-device-scale-factor=0.8

**Comment by rickswe** [comment]
I think this may be more of a wayland/chromium issue. I've noticed when I disable wayland in the flags
and enable x11 the problem goes away.
